docs/reference/connectors/gbq.md

```diff
@@ -26,14 +29,36 @@ The timeouts are in nanoseconds.
 |-----------------|------------------------------------------------------------------------------------------------------------------|
 | table_id        | The identifier of the table in the format: `projects/{project-name}/datasets/{dataset-name}/tables/{table-name}` |
 | connect_timeout | The timeout in nanoseconds for connecting to the Google API                                                      |
-| request_timeout | The timeout in nanoseconds for each request to the Google API                                                    |
+| request_timeout | The timeout in nanoseconds for each request to the Google API. A timeout hit will fail the event.                |
```

Comment:
Since people seem to struggle with this, let's highlight it?

```suggestion
| request_timeout | The timeout in **nanoseconds** for each request to the Google API. A timeout hit will fail the event.                |
```